cmd/gateway/azure/gateway-azure.go

```diff
@@ -983,10 +985,8 @@ func (a *azureObjects) DeleteObjects(ctx context.Context, bucket string, objects
 	dobjects := make([]minio.DeletedObject, len(objects))
 	for idx, object := range objects {
 		_, errs[idx] = a.DeleteObject(ctx, bucket, object.ObjectName, opts)
-		if errs[idx] == nil {
-			dobjects[idx] = minio.DeletedObject{
-				ObjectName: object.ObjectName,
-			}
+		dobjects[idx] = minio.DeletedObject{
```

Comment:
If a.DeleteObject() returns nil when the object is inexistant, then we don't have to disable this check.

(in S3, delete objects can return errors, but I know, not for inexistant objects)

Reply:
@kannappanr ^^

Reply:
The objectname needs to be set even when there is error, as this is being used in the handler